pkg/anypoint/client.go

```diff
@@ -349,6 +352,80 @@ func (c *AnypointClient) GetAnalyticsWindow() ([]AnalyticsEvent, error) {
 	return events, err
 }
 
+func (c *AnypointClient) GetSLATiers(apiId int64) (*Tiers, error) {
+	var slatiers Tiers
+	headers := map[string]string{
+		"Authorization": "Bearer " + c.auth.GetToken(),
+	}
+	url := fmt.Sprintf("%s/apimanager/api/v1/organizations/%s/environments/%s/apis/%d/tiers",
+		c.baseURL, c.auth.GetOrgID(), c.environment.ID, apiId)
+
+	request := coreapi.Request{
+		Method:      coreapi.GET,
+		URL:         url,
+		QueryParams: nil,
+		Headers:     headers,
+	}
+	err := c.invokeJSON(request, &slatiers)
+	return &slatiers, err
+}
+
+func (c *AnypointClient) CreateClientApplication(apiInstanceID string, app *AppRequestBody) (*Application, error) {
+	var application Application
+	query := map[string]string{
+		"apiInstanceID": apiInstanceID,
+	}
+
+	url := fmt.Sprintf("%s/exchange/api/v1/organizations/%s/applications", c.baseURL, c.auth.GetOrgID())
```

Comment:
Can this, 394, and 414 be combined to a getApplicationsURL method? I just noticed 394 is v2 vs v1 so maybe not that one.

Reply:
Actually the Mulesoft trial account uses v1 to create a new app and v2 to delete an app. Need to check this with Enterprise account whose account we've just got